src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -28,19 +29,30 @@ export type SurveyTaskListListPreviewState = "IncompleteTasks" | "CompleteTasks"
 
 export default function (props: SurveyTaskListProps) {
 	const [loading, setLoading] = useState(true);
-	const [tasks, setTasks] = useState<SurveyTask[] | null>(null);
+	const [tasks, setTasksInner] = useState<SurveyTask[] | null>(null);
+	const [activeSurveys, setActiveSurveys] = useState<string[]>([]);
```

Comment:
You should never be able to have more than 1 active survey...you can't launch them on top of each other

Reply:
Oh...but I guess in a very very slow loading scenario I see how this could happen